2_Clustering/1 Clustering Algorithms.ipynb

```diff
@@ -3869,38 +3869,109 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    clusterer = hdbscan.HDBSCAN(min_cluster_size=10, gen_min_span_tree=True)
</pre><p>generating the spanning tree is a quite costly operation. Try to set  <code>gen_min_span_tree=False</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WebValley2021ReImagined/unsupervised-learning/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>